wlog-glog/internal/marshalers/wtracing_spanmodel.go

```diff
@@ -68,6 +68,19 @@ func marshalWTracingSpanModel(key string, val interface{}) string {
 		}
 	}
 
+	if tags := span.Tags; tags != nil && len(tags) > 0 {
+		_, _ = builder.WriteString(`, tags:{`)
+		idx := 0
+		for k, v := range tags {
+			builder.WriteString(k + ":" + v)
+			if idx != len(tags)-1 {
+				builder.WriteString(separator)
+			}
+			idx = idx + 1
+		}
+		_, _ = builder.WriteString("}")
+	}
+
```

Comment:
wlog-glog seems to be completely broken and deserve some double checking of its current state. I wrote this addition to fit what it is currently producing. The biggest problem seems to be that it does not print double quotes. See this example output from its test:
```
type: trace.1, span: traceId: 13b9ae9acce1c990, id: 7b485c918531979b, name: testOp {parentId: 13b9ae9acce1c990, timestamp: 1613488524158033, duration: 0, tags:{key0:value0}}
```

Reply:
Thanks for flagging this. It makes sense to match the current output and then we can address any issues wholesale.